src/core/Coin.ts

```
@@ -69,7 +69,7 @@ export class Coin extends JSONSerializable<Coin.Amino, Coin.Data, Coin.Proto> {
   }
 
   public static fromString(str: string): Coin {
-    const m = str.match(/^(-?[0-9]+(\.[0-9]+)?)([0-9a-zA-Z/]+)$/)
+    const m = str.match(/^(-?[0-9]+(\.[0-9]+)?)([a-zA-Z][0-9a-zA-Z/:._-]{2,127})$/)
```

ESLint

[error] Replace `/^(-?[0-9]+(\.[0-9]+)?)([a-zA-Z][0-9a-zA-Z/:._-]{2,127})$/` with `⏎······/^(-?[0-9]+(\.[0-9]+)?)([a-zA-Z][0-9a-zA-Z/:._-]{2,127})$/⏎····`

(prettier/prettier)